logger.go

```diff
@@ -0,0 +1,6 @@
+package rsync
+
+import "github.com/gokrazy/rsync/internal/log"
+
+// Logger logs messages.
```

Comment:
The current comment doesn’t really tell the reader much :)

How about:

Logger is an interface that allows specifying your own logger. By default, the Go log package is used, which prints to stderr.

Reply:
Updated, PTAL

Reply:
You wrote “updated”, but the message still reads the same to me?

Reply:
I blame for github PR review's UI... it's showing the updated doc string in the latest file: https://github.com/gokrazy/rsync/pull/20/files#diff-a06377f540cbeeb0aee48171c155ff0a45ece25e91e19c06a8039f8e8f16e7bc

Reply:
Nah, it’s not the UI — you changed internal/log.Logger, I was commenting on rsync.Logger :)

Anyway, I fixed this in the `main` branch now :)

Reply:
Oh cool, thanks for the fix!